README.md

```diff
@@ -34,7 +41,28 @@ This code requires Python 3.6 or a higher version. If you haven’t already, dow
 ```
 python icone_translator.py -i inputfile.xml -o outputfile.geojson
 ```
+Example usage:
+```
+python icone_translator.py -i '../sample files/icone data/incidents_extended.xml' 
+```
+
+### Testing
+
+#### Step 1: Run the unit test for translator script (from root directory)
+
+```
+coverage run -m pytest 'test/' -v 
+```
+### Google Cloud Function
+
+A system was created in google cloud platform to automatically translate iCone data to WZDx message.This system consits of two buckets and a cloud function.
+When a file is uploaded to the first bucket, the cloud function automatically downloads the file, translates to a WZDx message and uploads it to the second bucket.
+
+![alt text](https://github.com/CDOT-CV/Work_Zone/blob/dev/translator/GCP_cloud_function/iCone%20Translator%20block%20diagram.png)
+
+### Documentation
 
+documentation for iCone to WZDx translator is located here: https://github.com/CDOT-CV/Work_Zone/tree/main/translator/docs
```

Comment:
As above, this should  be a relative link so we avoid issues pointing to a specific branch that may or may not have documents.